ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/workloadTableToolbar.test.js

```diff
@@ -0,0 +1,48 @@
+import withAuth from '../../../helpers/basicAuth';
```

Comment:
This test should really be a component level test, either with Jest or with Cypress component testing (TBD - needs some experimentation). In the interest of getting feature coverage though it can be an e2e test that maybe we can convert as a follow up.